src/main/webapp/scripts/list-video-service.js

```diff
@@ -0,0 +1,24 @@
+YOUTUBE_API_ERROR = new Error('Oops! We were not able to get the videos from Youtube')
+
+function getTrendingFromYoutubeApi() {
+  // Ready in a later PR: let user choose country, currently defaulting to US
+  return gapi.client.youtube.videos.list(getPopularRequest('US'))
+  .then((response) => {
+    return response.result;
+  })
+  .catch((error) => {
+    console.log('An error occurred when making API list request:', error);
+    throw YOUTUBE_API_ERROR;
```

Comment:
Nit: This catch is not adding any new information that `error` probably has. If error is descriptive enough then you don't need to catch it.